src/Commands/RunScan.ts

```diff
@@ -211,3 +213,15 @@ export class RunScan implements CommandModule {
     }
   }
 }
+
+async function fetchTestTypes(baseURL: any): Promise<string[]> {
```

Comment:
Avoid expecting anything without any reason, especially when you can easily declare strict types